src/components/ActionSheet.js

```diff
@@ -0,0 +1,64 @@
+import React, {useEffect, useRef} from 'react';
+import ActionSheet from 'react-native-actionsheet';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
+import {showToast} from '../utils/LocalStorage';
+import {checkStorageReadPermission} from '../utils/PermissionUtils';
+
+export const FilePickerActionSheet = ({
+  onPickFile,
+  setDisplayActionSheet,
+  setShowActionSheet,
+  mediaType,
+}) => {
+  const refActionSheet = useRef(null);
+  const actions = [launchCamera, launchImageLibrary];
+
+  useEffect(() => {
+    // Start showing action sheet as soon as the ref is active
+    if (refActionSheet.current) {
+      refActionSheet.current.show();
+    }
+  }, [refActionSheet]);
+
+  const onPressAction = async index => {
+    if (mediaType === 'image') {
+      try {
+        if (await checkStorageReadPermission()) {
+          const {
+            assets: [imagePickerResponse],
+          } = await actions[index]?.();
+          onPickFile(imagePickerResponse);
+        }
+      } catch (error) {
+        setShowActionSheet(false);
+
+        setDisplayActionSheet(false);
+        console.log('Upload error: ', error);
+        showToast(`Upload asset error: ${JSON.stringify(error)}`);
+      }
+    } else {
+      try {
+        if (await checkStorageReadPermission()) {
+          const {
+            assets: [imagePickerResponse],
+          } = await actions[index]?.({mediaType: 'video', includeBase64: true});
```

Comment:
@Charan017 Why are we using `includeBase64: true` here? Are we using file's base64 string anywhere during our processing?

Reply:
By removing `includeBase64` it's working fine. Actually it is not required.